docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,231 +9,1402 @@
    },
```

Comment:
<p>The initial underscore is part of the target definition syntax in rst. The referenced target should be <code>numpy:basics.broadcasting</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>